BusyList/Commands/Commands.cs

```diff
@@ -4,7 +4,7 @@
 
     public record ReadCommand(int Id) : Command;
     public record NextCommand() : Command;
-    public record AddCommand(string Description) : Command;
+    public record AddCommand(string Description, PriorityEnum? Priority = null) : Command;
```

Comment:
Shouldnt the default be Low - considering its low prioritized. A todo will always have a priority, and if we simply dont care it means its low prioritized

Reply:
Interesting take. If we go with "all items are prioritized" then I suggest we rename "medium" to "normal" and have that be our default. It makes sense to be able to make tasks that are explicitly lower or higher than the default.

Reply:
So, should I change it? 😁

Reply:
If u want

Reply:
As the guy who both suggested and implemented the feature, I think its up to you to decide. If we decide to have "no priority" be the default, how will it compare to a low, a medium and a high one?